otx/algorithms/common/adapters/mmcv/nncf/runners.py

```diff
@@ -139,11 +139,14 @@ def validation_fn(self, *args, **kwargs):  # pylint: disable=unused-argument
         Evaluation is assumed to be already done at this point since EvalHook was called.
```

Comment:
Shouldn't the docstring be changed?

Reply:
This line is removed in [`0d02dfe` (#1574)](https://github.com/openvinotoolkit/training_extensions/pull/1574/commits/0d02dfed42e304c9d096f958e55053b226e6fde5).